components/webui/imports/ui/SearchView/SearchResults/SearchResultsTable/index.jsx

```diff
@@ -77,6 +78,12 @@ const SearchResultsTable = ({
         }
     };
 
+    const handleSearchResultClick = (ev) => {
+        const {orig_file_id: origFileId, log_event_ix: logEventIx} = ev.currentTarget.dataset;
+        window.open(`${Meteor.settings.public.LogViewerWebuiClientUrl
```

Comment:
In the future, we can open the Log Viewer in a split pane so users can see both the CLP search results and the log context at the same time. For the MVP, we can open the Log Viewer WebUI client in a new tab. Feel free to object.